examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb

```diff
@@ -3,7 +3,7 @@
   {
```

Comment:
<p>The name of the notebook should be <code>Serving-Ranking-Models-With-Merlin-Systems.ipynb</code> .</p><p> </p><p>I'd rephrase as:</p><p> </p><p> <code>... notebook that deploys an ensemble of&nbsp;<a href="https://github.com/NVIDIA-Merlin/NVTabular" target="_blank">NVTabular</a>&nbsp;workflow and a trained model from Merlin Models to ...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>